local_modules/cashweb/lib/relay/constructors.ts

```diff
@@ -10,6 +10,7 @@ import { PayloadConstructor } from './crypto'
 import VCard from 'vcf'
 import { AuthWrapper } from '../auth_wrapper/wrapper_pb'
 import { Wallet } from '../wallet'
+import { TextEncoder } from 'util'
```

Comment:
I'm suspicious of this. I had weird type stuff with this before. What was the issue that needed it?